src/utils/general.ts

```diff
@@ -894,8 +896,9 @@ export function waitForPageToBeVisible() {
 export async function clearCache() {
   const cacheArray = await api.storage.list();
   let deleted = 0;
+  if (api.type === 'webextension') localStore.clear();
 
-  j.$.each(cacheArray, function (index, cache) {
+  j.$.each(cacheArray, (index, cache) => {
```

Comment:
Just as an hint. This can be dangerous with jquery loops, as they often rely on `this`.

In this case it is fine. Specially as it make no sense that it is a jquery loop to beginn with instead of a normal js loop.

Reply:
Thanks. When I first saw this I couldn't understand what it this loop